### ISSUE TITLE
Current bus stop name not showing

### ISSUE THREAD

**Issue by brokenphilip** [brokenphilip/OMSIPresence#6]
When hovering over the "delay" small icon in your Discord Rich Presence, sometimes the current bus stop name (inside the parenthesis, before the delay itself) will not show. This is because the `AI_Scheduled_NextBusstopName` field of your `TRoadVehicleInst` (current vehicle instance) is not defined (a null `char` pointer)

When OMSIPresence tries to update your status every second, the following warning is printed to the `logfile.txt`:
```
[OMSIPresence] TRoadVehicleInst.AI_Scheduled_NextBusstopName was null
```

This warning used to be pretty rare until I decided to play the [Novi Sad](https://www.youtube.com/watch?v=5eDawD-y5Ws) map, especially longer lines that go outside the city and usually have very distant bus stops. As can be seen using the `Shift+Y` debug-red-text-print-thing, distant bus stops have a bug where your timetable delay according to them is incorrectly calculated. In normal circumstances, the delay takes into account your distance (and, consequently, your speed), which results in the delay making very little to no changes for a well-defined timetable profile (if you're expected to drive 50km/h between each stop, and you do exactly that, the delay will remain virtually unchanged). However, distant bus stops do not have this calculation, resulting in the delay ticking up every second (as if you're not moving at all) until you get sufficiently close to them, when it suddenly fixes itself.

As it turns out, `AI_Scheduled_NextBusstopName` is `nullptr` when the bus stop is out of reach (ie. not spawned in yet) and it fixes itself when you get near the bus stop in the exact same way the delay gets fixed as well. The fix would be to use whatever the `Shift+Y` debug-red-text-print-thing uses to print the current bus stop, which (at a very quick glance because it's pretty late right now) I'm pretty sure just takes the information directly from the timetable (oh and also I may have found out that half of my `GetBusStopCount` code might be useless lol).

**Comment by brokenphilip** [comment]
Fixed as of 1.3. (forgot to close this a month ago lol, oops)

**Comment by brokenphilip** [comment]
While not necessarily related to the issue (especially considering it's been fixed), it is worth noting that these incorrect calculations do not occur when playing with the "Load whole map at start" option.
